File: server/datasets/controlled_benchmarks/python/multiple_files/multiple_e3/type3_composite_1_max_subarray.py
def get_max_sum(numbers):
    length = len(numbers)
    highest = numbers[length-1]
    temp_sum = numbers[length-1]
    
    dummy = [0] * 5
    
    for idx in range(length-2, -1, -1):
        if numbers[idx] > temp_sum + numbers[idx]:
            temp_sum = numbers[idx]
        else:
            temp_sum = temp_sum + numbers[idx]
            
        if temp_sum > highest:
            highest = temp_sum
            
    return highest

File: server/datasets/controlled_benchmarks/java/multiple_files/multiple_e3/type3_composite_1_max_subarray.java
public class Solution {
    public int get_max_sum(int[] numbers) {
        int length = numbers.length;
        int highest = numbers[length - 1];
        int temp_sum = numbers[length - 1];
        
        int[] dummy = new int[5];
        
        for (int idx = length - 2; idx >= 0; idx--) {
            if (numbers[idx] > temp_sum + numbers[idx]) {
                temp_sum = numbers[idx];
            } else {
                temp_sum = temp_sum + numbers[idx];
            }
            
            if (temp_sum > highest) {
                highest = temp_sum;
            }
        }
        return highest;
    }
}
